frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylist.tsx

```diff
@@ -119,8 +120,10 @@ export function SessionRecordingsPlaylist({
                     filterActions={
                         notebookNode ? null : (
                             <RecordingsUniversalFilters
+                                resetFilters={resetFilters}
                                 filters={filters}
                                 setFilters={setFilters}
+                                totalFiltersCount={totalFiltersCount}
```

Comment:
i'd still be tempted to make a "filtersHaveChanged" selector or something so that the logic is in the logic and we're not passing the count around